src/bot.js

```diff
@@ -41,19 +43,41 @@ try {
         // Our bot needs to know if it will execute a command
         // It will listen for messages that will start with `!`
         if (message.content.substring(0, 1) == '!') {
-            switch (message.content.toLocaleLowerCase()) {
+            // Split the input if it contains a space to support giving
+            // arguments to a command
+            command = message.content
+            if(command.indexOf(' ') != -1) {
+               command =  command.substring(0,command.indexOf(' '));
+            }
+            console.log(command);
+            command = command.toLocaleLowerCase()
+            switch (command) {
                 // !ping
                 case '!ping':
                     message.channel.send('pong');
                     break;
                 case '!help':
                     message.channel.send(`Currently supported commands
                     !pickGame: Suggests a game to play
+                    !myGames:  Set the games you want to play. Seperate each game by comma
+                    !deleteAllMyGames: Deletes user's game list from database
                     !online:   Messages all online users asking to join the channel
                     !pong:     Replies with 'pong'`);
                     break;
                 case '!pickgame':
-                    message.channel.send(gameChooser([true, true, true, true, true]))
+                    // Pick a game for the members in a voice channel
+                    const voiceChannel = message.member.voice.channel;
+                    if(!voiceChannel){
+                        return message.channel.send("Please join voice channel");
+                    }
+                    vcUsers = [];
+                    voiceChannel.members.each((user) =>{
+                      vcUsers.push(user.user.username);
+                    });
```

Comment:
This can be replaced with 
```suggestion
                    vcUsers = voiceChannel.members.map(user => user.user.username);
```